A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -59,10 +59,14 @@ var A320_Neo_UpperECAM;
     A320_Neo_UpperECAM.createRectangle = createRectangle;
     function createSlatParallelogram(_class, _cx, _cy) {
         const parElement = document.createElementNS(Avionics.SVG.NS, "polygon");
-        const _w = 5;
-        const _h = 6;
-        const _ox = 2;
-        const _oy = 1;
+        const _xOffset = 3.2;
+        const _yOffset = 1;
+        const _w = 3;
+        const _h = 4;
+        const _ox = 1.5;
+        const _oy = 1.5;
+        _cx = _cx + _xOffset;
```

Comment:
It's not a good idea to re-assign function parameters. You should create a new const instead.